### PR TITLE
Newcomers_Guide.rst : Add smart question link

### PR DESCRIPTION
**Pull request by yashraj-Dhanuka**
Newcomers_Guide.rst: Add smart question link

closes https://github.com/coala/coala/issues/4929

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by yashraj-Dhanuka** [comment]
@mridubhatnagar i did not understand what to do , please elaborate and guide me again.
Thankyou

**Comment by mridubhatnagar** [comment]
As of now it looks like this
>Newcomers_Guide.rst: Add smart question link

>This commit mentions a link of
>smart Question
>in newcomers guide.

>Closes #4929

Now, you can change it to 

>Newcomers_Guide.rst: Add smart question link

>Closes #4929

**Comment by mridubhatnagar** [comment]
Commit body is the description of the commit.

**Comment by yashraj-Dhanuka** [comment]
@mridubhatnagar Thanks for guiding. Please Review again.

**Comment by mridubhatnagar** [comment]
Check how to amend the commit message. 
You'll have to amend the commit message and then do a force push.

Check coala Git basics docs for reference.

**Comment by mridubhatnagar** [comment]
ack  1752c51

**Comment by yashraj-Dhanuka** [comment]
@mridubhatnagar Thank you so much for guiding me ,I am Literally very happy with my first successful contribution.Now should i move to newcomer/low tags ?

**Comment by mridubhatnagar** [comment]
@yashraj-Dhanuka :) Happy to help. So, the changes are approved. If maintainers find it good enough they'll merge it. Or in case they need this to be improved further they'll tell you accordingly. 
Yes, this issue is done now. You can move on to issues tagged as difficult/low. Get it assigned to yourself. 
And you can start working on it.

File: docs/Developers/Newcomers_Guide.rst [hunk 1]
```diff
@@ -459,8 +459,9 @@ go for `a low issue <https://coala.io/low>`__ - they are really rewarding!
 
 .. note::
 
-    If you need help picking up an issue, you can always ask us and we'll help
-    you!
+    If you need help picking up an issue, you can always
+    ask us,also keep in mind `Don't ask to ask,just ask <http://sol.gfxile.net/dontask.html>`_
```
Comment (Makman2) [inline docs/Developers/Newcomers_Guide.rst]: missing space after comma

Comment (Makman2) [reply docs/Developers/Newcomers_Guide.rst]: (2x)

Comment (Makman2) [inline docs/Developers/Newcomers_Guide.rst]: I would also replace the first comma with a period

**Comment by Makman2** [comment]
If you want to get your commit accredited correctly, please link the e-mail given in GitHub account to the commit :) (via the author information)

**Comment by Makman2** [comment]
unack 1752c51

**Comment by yashraj-Dhanuka** [comment]
@Makman2 Please have a look again .Thankyou  !!!

File: docs/Developers/Newcomers_Guide.rst [hunk 2]
```diff
@@ -459,8 +459,9 @@ go for `a low issue <https://coala.io/low>`__ - they are really rewarding!
 
 .. note::
 
-    If you need help picking up an issue, you can always ask us and we'll help
-    you!
+    If you need help picking up an issue.You can always
```
Comment (Makman2) [inline docs/Developers/Newcomers_Guide.rst]: Missing space after period.

File: docs/Developers/Newcomers_Guide.rst [hunk 3]
```diff
@@ -459,8 +459,9 @@ go for `a low issue <https://coala.io/low>`__ - they are really rewarding!
...
+    ask us, also keep in mind `Don't ask to ask, just ask <http://sol.gfxile.net/dontask.html>`_
```
Comment (Makman2) [inline docs/Developers/Newcomers_Guide.rst]: maybe remove `also`? Up to you :+1:

**Comment by Makman2** [comment]
You don't want to link your GitHub profile to your commit? ;)

**Comment by Makman2** [comment]
unack 9032578

**Comment by Makman2** [comment]
Closing due to inactivity.

**Pull request closed without merging**